dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java

```diff
@@ -30,6 +34,7 @@ public CollectionEntry(long id, Date dbUpdateDate, String entryTypeString, Sourc
         setId(id);
         setEntryPath(sourceControl.toString(), organization, repository, entryName);
         setVersionName(versionName);
+        //setLabels(labels);
```

Comment:
Avoid adding in code that is immediately commented and unlikely (?) to be uncommented

Reply:
Used to be 25 without it. And yeah guessing that number is the same as the number of entries